README.md

```diff
@@ -98,9 +98,13 @@ Register a factory function to create mock responses for each PATCH request to a
 
 Register a factory function to create mock responses for each DELETE request to a specific URL.
 
-#### .mock(method, url | regex, fn)
+#### .any(fn)
```

Comment:
Yeah, sorry, the docs probably aren't ideal here. Would this be clearer?
```
.mock(method: String|RegExp, url: String|RegExp, fn: Function)
.mock(fn: Function)
```

Lets leave out the alias for now.